treeherder/webapp/api/performance_data.py

```diff
@@ -64,7 +64,14 @@ def list(self, request, project):
                 framework__in=frameworks)
 
         interval = request.query_params.get('interval')
```

Comment:
Yes, exactly. :) I think the following options make sense:

1. interval. As currently implemented
2. start_date, end_date
3. just start_date

I would make the API return some kind of error if both interval + start date are defined (or there is an end_date but no start_date).